pkg/storage/blockdev.go

```diff
@@ -165,11 +167,163 @@ func GetBlockStats() ([]BlockDev, error) {
 		if err != nil {
 			return nil, err
 		}
-		blockdevs = append(blockdevs, BlockDev{Name: devname, Stat: *bstat})
+		devpath := path.Join("/dev/", devname)
+		uuid := getUUID(devpath)
+		blockdevs = append(blockdevs, BlockDev{Name: devname, Stat: *bstat, FsUUID: uuid})
 	}
 	return blockdevs, nil
 }
 
+// func getUUID(devpath string) (fsuuid string, err error) {
+// 	output, err := exec.Command("blkid", devpath).Output()
+// 	if err != nil {
+// 		return fsuuid, err
+// 	} else {
+// 		fields := strings.Fields(string(output))
+// 		for _, field := range fields {
+// 			if strings.Contains(field, "UUID=") && !strings.Contains(field, "PARTUUID=") {
+// 				fsuuid = strings.Trim(field, `UUID"=`)
+// 			}
+// 		}
+// 		return fsuuid, nil
+// 	}
+// }
+
+func getUUID(devpath string) (fsuuid string) {
```

Comment:
We can keep this for now, but I wonder if we could avoid hardcoding all this filesytem specific stuff by just reading everything in `/dev/disk/by-uuid` once and then doing a reverse matching with the mount point.

The main reason why I started to think about this is because I realized we also need `btrfs` here :D

Reply:
dev/disk/by-uuid is not available

Reply:
Awut? I think it should be :D

Anyway, this was more of a general comment for the future, we can keep this for now. But we will need `btrfs` support eventually.